### PR TITLE
Resolves GH-16: Switch off abandoned CLI parsing library to picocli

### PR DESCRIPTION
**Pull request by romeara**
Args4j is no longer being actively developed - switch to the current
widely-used library to continue receiving improvements and fixes

### CODE REVIEW

File: src/main/java/org/starchartlabs/lure/command/PostbinCommand.java [hunk 1]
```diff
@@ -47,6 +49,9 @@
  * @author romeara
  * @since 0.2.0
  */
+@Command(
+        description = "Pipes a payloads from postb.in to an indicated web URL, mimicing the GitHub webook call pattern",
```
Comment (nickavv) [inline src/main/java/org/starchartlabs/lure/command/PostbinCommand.java]: Think this is spelled mimicking

File: src/main/java/org/starchartlabs/lure/command/PushCommand.java [hunk 2]
```diff
@@ -38,13 +37,19 @@
...
+import picocli.CommandLine.ParentCommand;
 
 /**
  * Command line sub-command that will push raw data to a web URL following patterns defined for GitHub webhook calls
  *
  * @author romeara
  * @since 0.1.0
  */
+@Command(
+        description = "Pushes a provided payload to an indicated web URL, mimicing the GitHub webook call pattern",
```
Comment (nickavv) [inline src/main/java/org/starchartlabs/lure/command/PushCommand.java]: Same as above

**Review by nickavv — COMMENTED** [review]
Minor spelling comments but otherwise looks good

File: src/main/java/org/starchartlabs/lure/command/PostbinCommand.java (hunk 1 above)
Comment (romeara) [reply src/main/java/org/starchartlabs/lure/command/PostbinCommand.java]: good catch, fixed

File: src/main/java/org/starchartlabs/lure/command/PushCommand.java (hunk 2 above)
Comment (romeara) [reply src/main/java/org/starchartlabs/lure/command/PushCommand.java]: good catch, fixed

**Pull request merged**